client/dive-common/components/AttributeEditor.vue

```diff
@@ -128,14 +162,71 @@ export default defineComponent({
             required
           />
           <v-select
-            v-model="datatype"
+            :value="datatype"
             :items="[
               { text: 'Boolean', value: 'boolean' },
               { text: 'Number', value: 'number' },
               { text: 'Text', value: 'text' }
             ]"
             label="Datatype"
+            @change="typeChange"
           />
+          <div v-if="datatype=== 'number'">
+            <v-radio-group
+              :value="(editor && editor.type) || 'combo'"
+              row
+              label="Display Type:"
+              @change="numericChange"
+            >
+              <v-radio
+                label="Input Box"
+                value="combo"
+              />
+              <v-radio
+                label="Slider"
+                value="slider"
+              />
+            </v-radio-group>
+          </div>
+          <div v-if="datatype === 'number' && editor && editor.type === 'slider'">
+            <v-row class="pt-2">
+              <v-text-field
+                v-model.number="editor.range[0]"
+                dense
+                outlined
+                :step="editor.range[0]> 1 ? 1 : 0.01"
+                type="number"
+                label="Lower"
+                :max="editor.range[1]"
+                hint="Lower limit for slider"
+                persistent-hint
+              />
+              <v-text-field
+                v-model.number="editor.range[1]"
+                dense
+                outlined
+                :step="editor.range[1]> 1 ? 1 : 0.01"
+                type="number"
+                label="Upper"
+                :min="editor.range[0]"
+                hint="Upper limit for slider"
+                persistent-hint
+              />
+            </v-row>
+            <v-row class="pt-2">
+              <v-text-field
+                v-model.number="editor.steps"
+                dense
+                outlined
+                :step="editor.steps> 1 ? 1 : 0.01"
+                type="number"
+                label="Slider Resolution"
```

Comment:
In would change label to something like 'Slider Step Interval' and the hint to something like 'Slider will increase/decrease by this value'